tests/usdt_sample/scripts/latency.c

```diff
@@ -0,0 +1,127 @@
+#include <linux/blkdev.h>
+#include <uapi/linux/ptrace.h>
+
+// The following methods were taken from the verbose output of argdist.py.
+// Function names were adjusted to match the BPF function name expectations.
+static inline int _bpf_readarg_trace_operation_start_1(struct pt_regs* ctx, void* dest, size_t len)
+{
+    if (len != sizeof(uint64_t))
+        return -1;
+    {
+        u64 __addr = ctx->bp + (-24);
+        uint64_t __res = 0x0;
+        bpf_probe_read(&__res, sizeof(__res), (void*)__addr);
+        *((uint64_t*)dest) = __res;
+    }
+    return 0;
+}
+static inline int _bpf_readarg_trace_operation_start_2(struct pt_regs* ctx, void* dest, size_t len)
+{
+    if (len != sizeof(uint64_t))
+        return -1;
+    *((uint64_t*)dest) = (uint64_t)ctx->ax;
```

Comment:
The latest code base has "*(volatile uint64_t *)&ctx->ax" instead of "(uint64_t)ctx->ax" to prevent compiler optimization resulting programs being rejected by verifier. Could you update it?